### ISSUE TITLE
Inserting image links in Markdown format

### ISSUE THREAD

**Issue by mxro** [cloudy9101/obsidian-image-inserter#14]
Great plugin!

I noticed the plugin inserts image links using the wikilink format, such as:

```
![[Inserted image 20230312174133.png]]
```

There is a setting in Obsidian that controls if images should be inserted as wikilink or Markdown format:

![image](https://user-images.githubusercontent.com/1448524/224528766-6bb91778-8639-464b-950d-e5fca8c89798.png)

Would it be possible to change this plugin to respect that setting or add an additional option to this plugin to determine whether images are inserted in wiki or markdown format.

For reference, Markdown format would be:

```
![](image-path.png)
```

**Comment by cloudy9101** [comment]
Hi @mxro, that's a great idea. I had not realized there was a setting for the format.
Updated the plugin to respect this setting in https://github.com/cloudy9101/obsidian-image-inserter/releases/tag/0.5.2

**Comment by mxro** [comment]
Perfect! Thank you!

Does this work as well when 'Insert Mode' is 'Local'?

![image](https://user-images.githubusercontent.com/1448524/228950713-6e4bb10b-b27e-45ae-862e-a7481458de35.png)

Seems to me it is still inserting links in the wikilink format for me when using this setting (But when Insert Mode is Remote it works).

**Comment by cloudy9101** [comment]
I think we can't do that in markdown format for local images, it can't be rendered correctly in Obsidian. I'm not quite sure, correct me if I'm wrong.

**Comment by mxro** [comment]
Obsidian displays images defined in Markdown with the Markdown image syntax without problems, even when referencing local files!

E.g. an image defined like this is displayed correctly in the editor:

```markdown
![Screenshot of React UI that shows a sign in and register buttons](images/Pasted%20image%2020221226160435.png)
```

**Comment by cloudy9101** [comment]
Hi @mxro, you were correct. I had misremembered. The `wikilinks` formatting cannot be used with remote images. The issue with local images not following the `wikilinks` setting was due to a bug. I have now fixed it and released a new version, [0.5.3](https://github.com/cloudy9101/obsidian-image-inserter/releases/tag/0.5.3). Please update and try again.

**Comment by mxro** [comment]
Perfect, thank you @cloudy9101 - it works now!
